docs/source/guides/models.mdx

```diff
@@ -201,10 +201,10 @@ with torch.inference_mode():
 Optimum extends 🤗`Diffusers` to support inference on Neuron. To get started, make sure you have installed Diffusers:
 
 ```bash
-pip install diffusers>=0.17.0
+pip install diffusers>=0.20.0
```

Comment:
should we add an optional with [diffusers] to the setup.p?

Reply:
We already have diffusers as a soft dependency in `setup.py`, let me update the doc using `optimum[neuronx, diffusers]` extra here.